op-chain-ops/genesis/config.go

```diff
@@ -31,6 +31,16 @@ var (
 	ErrInvalidImmutablesConfig = errors.New("invalid immutables config")
 )
 
+// todo(maurelian): Is it possible to reuse the type def in the superchainconfig.go bindings?
+// SequencerKeys is an auto generated low-level Go binding around an user-defined struct.
+type SequencerKeys struct {
+	BatcherHash       common.Hash
+	UnsafeBlockSigner common.Address
+}
+
+// SequencerKeysArray is an array of SequencerKeys.
+type SequencerKeysArray []SequencerKeys
```

Comment:
nit on the naming here, I don't love calling things based on implementation details, ie the array suffix in this case. Perhaps we can use the bindings type or rename the above to `SequencerKey` (no s) or `SequencerKeyPair` (then we are coupled to 2 keys which i think is fine for now) or `SequencerIdentity` but perhaps it should match what is in solidity, i just want to not use the type `SequencerKeysArray` and rename it to something like `SequencerKeys` (with the plural) or `SequencerSet`. This begs the question, the solidity allows for a set where implementing as a slice allows for duplicates so its not technically a set

Reply:
I don't think it is worth the complexity to worry about implementing it as a set, just adds extra complexity to json parsing

Reply:
Addressed this in the parent PR.